### DISCUSSION TITLE
GitHub Pages not serving images?

### DISCUSSION THREAD

**Discussion by JoFrhwld** [community/community#46149] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hello!

I have a GitHub organization, and I’ve set up an organization page repository for it. The problem I’m having is that recently images on the site aren’t being served. If you try to look at this png, for example, you get the site 404: 
https://lingmethodshub.github.io/content/R/vowel-plots-tutorial/index_files/figure-html/unnamed-chunk-4-1.png

But, the png *does* exist in the gh-pages branch: 
https://github.com/LingMethodsHub/LingMethodsHub.github.io/blob/gh-pages/content/R/vowel-plots-tutorial/index_files/figure-html/unnamed-chunk-4-1.png

My only thought about why this might be is that we’re being limited by a cap on org page images I wasn’t aware of?

**Comment by JoFrhwld** [comment, marked as answer]
For additional information, I've double checked the size of the gh-pages branch, and it's ~89 MB, so well below the 1 GB limit on pages.

**Reply by yoannchaudet** [reply]
In the settings for your site, you set the source branch to `main` and `/docs`, not `gh-pages` and `/docs`. Hence the 404.

**Reply by JoFrhwld** [reply]
Thanks! After finally digging into it today, in switching over to publishing from `gh-pages` to `/docs`, I'd forgotten to take `*_files` out of my `.gitignore`!

**Comment by ghost** [comment]
Just tried it and Yeah, its not working for me too (never noticed 🤔)
However, you could use a free image hosting platform like [imgur](https://imgur.com) or [Google Photos](photos.google.com) and embed the image using a url
Hope this helps ✌️

**Reply by ghost** [reply]
It exclusively doesn't work for `githubpages.com`  links
